### PR TITLE
Switchwire feed slot with pass-through

### PR DESCRIPTION
**Pull request by Jasrags**
* [X] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [X] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [X] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [X] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [X] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [X] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [X] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [X] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [X] Submitted STL files are oriented and scaled properly for printing.
  * [X] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

I wanted a feed slot for the SW that did not have the large hole and did not use VHB. 

#### Which mods/configurations/profiles are added by this PR?
A SW feed slot with a small pass-through hole

#### How was it tested? 
I printed it out and tested on my SW, works as expected

#### Screenshots 
Mounted_1
Mounted_2
s
CAD_offset_profile

#### Further note
My original plan was for a bowden tube but that did not work at high Z.

### CODE REVIEW

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

Please remove spaces from all filenames.

The bottom of this hole has too steep overhangs due to the fillet, please replace the fillet on that side with a 45 degree chamfer
Screen Shot 2021-10-02 at 14 01 21

File: printer_mods/README.md [hunk 1]
```diff
@@ -124,6 +124,7 @@ like so:
 |Iakabos| [c920_Mount](./Iakabos/C920_mount)|Slim mount for logitech c920 webcam|:x:|:heavy_check_mark:|:heavy_check_mark:|:x:|
 | | [V0 beltchains](./Iakabos/V0_beltchain)|Modified beltchain parts for V0 Z cables|:heavy_check_mark: |:x: |:x: |:x:|
 |jasonc |[V2.4 Z-Idler Shoulder Bolt Mod](./jasonc/Z_Idler_Shoulder_Bolt) |Modification to the idler carriers to accept a 25mm shoulder bolt. Ideal for Gates idlers. |:x: |:x: |:heavy_check_mark: |:x:|
+| Jasrags | [Feed slot with filament pass-through](./Jasrags/sw_fee_slot_filament_passthrough) | A feed slot with a filament pass-through | :x: | :x: | :x: | :heavy_check_mark: |
```
[the remark this answers is not in the document]
Comment (Jasrags) [reply printer_mods/README.md]: Done, Guess i made my branch before Trident, did not have that column until i pulled in upstream.

[the remark this answers is not in the document]
Comment (Jasrags) [reply printer_mods/README.md]: Other changes made as well. thanks

**Review by KiloQubit — APPROVED** [review]
Thanks for the update!

**Pull request merged**